### DISCUSSION TITLE
Non-core stable version

### DISCUSSION THREAD

**Discussion by glucaci** [open-telemetry/opentelemetry-dotnet#3239] [General]
There is planed to release a non-core stable version which correspond with core 1.2.0 ?

Currently there is no non-core version which reference the core 1.2.0.

Non-core 1.0.0-rc9.2 is using core 1.2.0-rc5
Non-core 1.0.0-rc9.3 is using core 1.3.0-beta.1

Thanks!

**Comment by tomkerkhove** [comment]
There is no ETA for it, see https://github.com/open-telemetry/opentelemetry-dotnet/issues/3203

**Comment by cijothomas** [comment]
https://github.com/open-telemetry/opentelemetry-dotnet/milestones Please check milestones to know about stable release date, and why some packages are not likely to be stable anytime soon.
